code/modules/hydroponics/goon/herb_seeds.dm

```diff
@@ -0,0 +1,277 @@
+	chems = list("charcoal" = list(1,10))
+
+	lifespan = 25
+	maturation = 6
+	production = 6
+	yield = 5
+	potency = 20
+
+/datum/seed/venne/toxic
+	name = "toxicvenne"
+	seed_name = "black venne"
+	display_name = "black venne"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/toxicvenne)
+	mutants = null
+	packet_icon = "seed-toxicvenne"
+	plant_icon = "toxicvenne"
+	chems = list("napalm" = list(1,10))
+
+/datum/seed/venne/curative
+	name = "curativevenne"
+	seed_name = "sunrise venne"
+	display_name = "sunrise venne"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/curativevenne)
+	mutants = null
+	packet_icon = "seed-curativevenne"
+	plant_icon = "curativevenne"
+	chems = list("oculine" = list(1,10), "mannitol" = list(1,10), "mutadone" = list(1,10))
+
+//Asomna/varieties.
+/datum/seed/cannabis
+	name = "cannabis"
+	seed_name = "cannabis"
+	display_name = "cannabis"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/cannabis)
+	mutants = list("omegaweed", "lifeweed", "deathweed", "rainbowweed")
+	packet_icon = "seed-cannabis"
+	plant_icon = "cannabis"
+	harvest_repeat = 1
+	chems = list("thc" = list(1,10))
+
+	lifespan = 25
+	maturation = 6
+	production = 6
+	yield = 5
+	potency = 20
+
+/datum/seed/cannabis/life
+	name = "lifeweed"
+	seed_name = "life weed"
+	display_name = "life weed"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/lifeweed)
+	mutants = null
+	packet_icon = "seed-lifeweed"
+	plant_icon = "lifeweed"
+	chems = list("omnizine" = list(1,10))
+
+/datum/seed/cannabis/death
+	name = "deathweed"
+	seed_name = "death weed"
+	display_name = "death weed"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/deathweed)
+	mutants = null
+	packet_icon = "seed-deathweed"
+	plant_icon = "deathweed"
+	chems = list("cyanide" = list(1,10))
+
+/datum/seed/cannabis/rainbow
+	name = "rainbowweed"
+	seed_name = "rainbow weed"
+	display_name = "rainbow weed"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/rainbowweed)
+	mutants = null
+	packet_icon = "seed-rainbowweed"
+	plant_icon = "rainbowweed"
+	chems = list("mindbreaker_toxin" = list(1,10))
+
+/datum/seed/cannabis/omega
+	name = "omegaweed"
+	seed_name = "omega weed"
+	display_name = "omega weed"
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/omegaweed)
+	mutants = null
+	packet_icon = "seed-omegaweed"
+	plant_icon = "omegaweed"
+	chems = list("LSD" = list(1,10),"suicider" = list(1,10),"space_drugs" = list(1,10),"mercury" = list(1,10),"lithium" = list(1,10),
+	"atropine" = list(1,10), "ephedrine" = list(1,10), "impedrezene" = list(1,10),"methamphetamine" = list(1,10),"THC" = list(1,10),"capsaicin" = list(1,10),"psilocybin" = list(1,10),"hairgrownium" = list(1,10),
+	"ectoplasm" = list(1,10),"bathsalts" = list(1,10),"itching_powder" = list(1,10),"crank" = list(1,10),"krokodil" = list(1,10),"catdrugs" = list(1,10),"histamine" = list(1,10))
```

Comment:
LSD = mindbreaker (for now until we have reagent decals and add in space fungsus on walls)
impedrezene = haloperidol.

Don't have ectoplasm or cat drugs yet.